python_ta/debug/snapshot.py

```diff
@@ -48,48 +50,82 @@ def snapshot():
     return variables
 
 
-def snapshot_to_json(snapshot_data: list[dict]) -> list[dict]:
+def snapshot_to_json(snapshot_data: List[Dict]) -> List[Dict]:
```

Comment:
Looking at the change here suggests an issue: you should avoid importing `List` (and `Dict`), and just use the default type names. The `__future__` import makes this compatible with older versions of Python.